package.json

```
@@ -21,7 +21,7 @@
     "server": "npm run view",
     "build": "node auspice.js build --verbose",
     "prepare": "npm run build",
-    "lint": "eslint .",
+    "lint": "eslint . --max-warnings=0",
```

_non-blocking nit_

be consistent with the other `eslint` rule...
```suggestion
    "lint": "eslint --max-warnings=0 .",
```

---

Done: https://github.com/nextstrain/auspice/blob/81bee2fdc6500168ddd1755085a89f3268d685d9/package.json#L24

I merged https://github.com/nextstrain/nextstrain.org/pull/1007 before seeing this, which applies in that repo too. I'll make a separate PR to fix the order there.